### PR TITLE
move to development and fix date

### PR DESCRIPTION
**Pull request by jdorweiler**
Move IA pages to development when we see the IA page link in a PR. 
Fix date bug. Localtime months start at 0 

@jbarrett

### CODE REVIEW

**Comment by jbarrett** [comment]
Do we need to make changes to the current data to compensate for the month issue?

**Comment by jdorweiler** [comment]
Maybe.  Let me check to see which IA pages had their date set that way.

**Comment by jdorweiler** [comment]
@jbarrett  I added some sql to fix that too.

File: script/ghIssues.pl [hunk 1]
```diff
@@ -140,12 +140,15 @@ sub getIssues{
                 } )->hri->one_row;
                 my $new_ia = 1 if !$ia;
 
+                # no auto generating IA pages from a PR anymore
+                return unless $ia;
+
                 my @time = localtime(time);
-                my $date = "$time[4]/$time[3]/".($time[5]+1900);
+                my ($month, $day, $year) = ($time[4]+1, $time[3], $time[5]+1900);
```
Comment (MariagraziaAlastra) [inline script/ghIssues.pl]: Pages to test in order to see whether the dates are parsed correctly:
- Live IA Page with issues (eg ia/view/calculator)
- Overview (ia/dev)
- Live Issues (ia/dev/issues)
- Dev Pipeline (ia/dev/pipeline)

**Comment by jdorweiler** [comment]
Fixed another date bug in `InstantAnswer.pm` too.

**Pull request merged**